src/main/java/core/basesyntax/service/DataParserService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface DataParserService<T> {
```

Comment:
When you have 'er' in name of class/interface it's doesn't really makes sense to add Service to name
```suggestion
public interface DataParser<T> {
```

Reply:
not fixed